internal/legacy/keystore/keystore_test.go

```diff
@@ -172,34 +172,34 @@ func TestChaining(t *testing.T) {
 			cc:             []byte("3318959fff419ab8b556facb3c429a86"),
 			origPrivateKey: []byte("5ffc975976eaaa1f7b179f384ebbc053"),
 			nextPrivateKeyUncompressed: []byte{
-				0xd3, 0xfe, 0x2e, 0x96, 0x44, 0x12, 0x2d, 0xaa,
-				0x80, 0x8e, 0x36, 0x17, 0xb5, 0x9f, 0x8c, 0xd2,
-				0x72, 0x8c, 0xaf, 0xf1, 0xdb, 0xd6, 0x4a, 0x92,
-				0xd7, 0xc7, 0xee, 0x2b, 0x56, 0x34, 0xe2, 0x87,
+				0x70 ,0x7b ,0x46 ,0x99 ,0x09 ,0x09 ,0x4f ,0x97,
```

Comment:
why we need to change priv/pub/sign etc related code to fix the unit-test?  I am not sure why it required for the master branch?